### PR TITLE
[chore] create supabase client

### PR DESCRIPTION
**Pull request by celinechoiii**
[//]: # "Feel free to customize this template to your project's vibes"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

- created supabase client in `api/supabase/createClient.ts`
- added `dotenv` and `@supabase/supabase-js` packages
- edited the PR template and README.md
- added the event schema

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- add all schemas in `types/schema.d.ts`

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"
https://github.com/calblueprint/trap-garden/pull/3

CC: @insert pl github username here

### CODE REVIEW

File: types/schema.d.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  event_id: UUID;
+  facility_id: UUID;
+  start_date_time: string; //timestamptz
+  end_date_time: string; //timestamptz
+  type_of_act: TypeOfAct;
+  genre: Genre;
+  needs_host: boolean;
+  performer_type: string;
+  event_status: EventStatus;
+}
```
Comment (pragyakallanagoudar) [inline types/schema.d.ts:45]: for clarification, will the other types (like `Facility`, `Volunteer`, etc.) be added in future sprints?

Comment (celinechoiii) [reply types/schema.d.ts:45]: Yes, the rest of the schema will be added in later sprints!
